beacon-chain/rpc/nodev1/node.go

```diff
@@ -156,7 +156,7 @@ func (ns *Server) ListPeers(ctx context.Context, req *ethpb.PeersRequest) (*ethp
 		stateIds = peerStatus.All()
```

Comment:
Behavior is the same but the reason this is changed is because the old code was using `go_grpc_gateway_library` as a bazel target when it should have been using `go_default_library`